utils/zerovec/src/map/borrowed.rs

```diff
@@ -220,10 +220,8 @@ where
 
 impl<'a, K, V> ZeroMapBorrowed<'a, K, V>
 where
-    K: ZeroMapKV<'a>,
-    V: ZeroMapKV<'a>,
-    K: ?Sized,
-    V: ?Sized,
+    K: ZeroMapKV<'a> + ?Sized + Ord,
```

Comment:
There was no `Ord` here before

```suggestion
    K: ZeroMapKV<'a> + ?Sized,
```